helm/templates/statefulset.yaml

```diff
@@ -29,6 +29,28 @@ spec:
         ports:
         - containerPort: {{ .Values.service.targetPort }}
           protocol: TCP
+        {{- if .Values.livenessProbe.enabled }}
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: {{ .Values.service.targetPort }}
+          initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.livenessProbe.successThreshold }}
+          failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+        {{- end }}
+        {{- if .Values.readinessProbe.enabled }}
+        readinessProbe:
+          httpGet:
+            path: /
```

Comment:
Referred the example here: https://github.com/helm/charts/blob/master/stable/vault-operator/templates/deployment.yaml#L55, here the path should be `/healthz`?

Reply:
Hi @mallow111  Thanks for your review. I updated the path from `/` to `/healthz`.